src/pandas_profiling/config.py

```diff
@@ -166,6 +166,9 @@ def clear(self):
         self.config = confuse.Configuration("PandasProfiling", __name__, read=False)
         self.set_file(str(get_config_default()))
 
+    def set_spark(self):
+        self.set_file(str(get_config("config_default_spark.yaml")))
```

Comment:
What is the advantage of creating an additional config file over using `config_default.yaml` for both engines (since the parameters for spark and pandas in the config are split already)? Knowing which engine should be used would be enough.

Reply:
oh i thought you previously didn't want separate parameters for each engine in https://github.com/pandas-profiling/pandas-profiling/pull/670#discussion_r557574342?

Reply:
ah ok, rereading your comment again, realised I misread it. I think it might be good to have different default values though for each engine (pandas does all by default, spark only does pearson for speed), and that logic has to be stored somewhere. Wondering if the config file a good place to store this? if not it'll have to be embedded in the code.